theano/tensor/opt.py

```diff
@@ -166,7 +166,7 @@ def scalarconsts_rest(inputs, elemwise=True, only_process_constants=False):
     return consts, origconsts, nonconsts
 
 
-def broadcast_like(value, template, fgraph, dtype=None):
+def broadcast_to(value, template, fgraph, dtype=None):
```

Comment:
Sorry for the confusion, but #164 is effectively concerned with making `broadcast_like` use the `Op` [`theano.tensor.extra_ops.broadcast_to`](https://github.com/pymc-devs/Theano-PyMC/blob/99f08ee3307b6084e204fa8327824af62e51b39a/theano/tensor/extra_ops.py#L1608) instead of the potentially expensive [`alloc`](https://github.com/pymc-devs/Theano-PyMC/blob/99f08ee3307b6084e204fa8327824af62e51b39a/theano/tensor/opt.py#L193) that it currently uses.

Reply:
> should I need to change the broadcast_to to broadcast_like in line no. 169.

Yes, you'll need to change the name back to `broadcast_like` to avoid a name clash with the existing `broadcast_to` function.